docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -66,6 +66,9 @@ abstract class Client {
     onSign: (message: string) => Promise<Cacao.Signature>
   }): Promise<string>;
 
+  // deletes all subscriptions, decouples client from the registered account and pauses watchSubscription requests
```

Comment:
I don't think deleting all subscriptions makes sense to do here: https://walletconnect.slack.com/archives/C044SKFKELR/p1695827130071699?thread_ts=1695826302.572449&cid=C044SKFKELR